tests/Oryx.Integration.Tests/Php/PhpGreetingsAppTest.cs

```diff
@@ -12,7 +12,6 @@
 
 namespace Microsoft.Oryx.Integration.Tests
 {
-    [Collection("Php integration")]
```

Comment:
Removed these from the tests, now that I've found that they can run in parallel as long as we give them some more time to start up, which I did with the changes that also check more often for container existence

Reply:
If we remove "php integration" collection from these tests, would they show up in the php integration jobs for nightly build or CI build? Just want to double check they won't removed from the pipeline jobs if we remove this line.

Reply:
Yes they will show up in the integration tests, since the tests themselves have the correct categories. The only thing that this collection was doing was making sure that the php tests were not running in parallel, which was a temporary fix for some transient failures we were seeing.